src/app/title/page.tsx

```diff
@@ -209,16 +263,62 @@ const DependencyOrder: React.FC<DependencyOrderProps> = ({ title }) => {
 	);
 };
 
+interface SequentialOrderProps {
+	title: SequentialOrderTitleDto;
+}
+
+const SequentialOrder: React.FC<SequentialOrderProps> = ({ title }) => {
+	const { previous, next } = title;
+
+	return (
+		<SequentialOrderBlock>
+			<SequentialAdjacent alignRight={true}>
+				{previous && (
+					<>
+						<h2>previous</h2>
+						<Poster src={previous.smallPosterUrl} alt={previous.name} />
+						<h3>{previous.releasedAtUtc?.getFullYear()}</h3>
+					</>
+				)}
+			</SequentialAdjacent>
+			<SequentialItem>
+				<LeadPoster src={title.largePosterUrl} alt={title.name} />
+			</SequentialItem>
+			<SequentialAdjacent alignRight={false}>
+				{next && (
+					<>
+						<h2>next</h2>
+						<Poster src={next.smallPosterUrl} alt={next.name} />
+						<h3>{next.releasedAtUtc?.getFullYear()}</h3>
+					</>
+				)}
+			</SequentialAdjacent>
+		</SequentialOrderBlock>
+	);
+};
+
 export default function Title() {
-	const dummyTitle: DependencyOrderTitleDto = {
+	const [orderType, setOrderType] = useState<"sequential" | "relational">(
+		"relational",
+	);
+
+	function cycleOrderType() {
+		setOrderType(orderType === "sequential" ? "relational" : "sequential");
+	}
```

Comment:
Typically I like to use arrow functions inside components.

```suggestion
	const cycleOrderType = () => {
		setOrderType(orderType === "sequential" ? "relational" : "sequential");
	}
```